endpoints/server_test.go

```diff
@@ -229,6 +239,10 @@ func TestServerServeHTTP(t *testing.T) {
 		{"POST", "TestDefault", `{"age": 20}`, ``, http.StatusOK},
 		{"POST", "TestDefault", `{"weight": 3.14}`, ``, http.StatusOK},
 		{"POST", "TestDefault", `{"name":"francesc", "age": 20}`, ``, http.StatusOK},
+		{"POST", "TestMinMax", `{"age":10,"weight":5,"grade":"C"}`, ``, http.StatusOK},
+		{"POST", "TestMinMax", `{"age":123,"weight":5,"grade":"C"}`, ``, http.StatusBadRequest},
+		{"POST", "TestMinMax", `{"age":10,"weight":1,"grade":"C"}`, ``, http.StatusBadRequest},
+		{"POST", "TestMinMax", `{"age":10,"weight":5,"grade":"G"}`, ``, http.StatusBadRequest},
```

Comment:
No, because age:0 will be considered as a missing field.

I'm not sure we can fix this as there's no way to make the difference between an unset int and an int set to 0.

The solution would be to use *int as protobufs do ... but that would make the code so much worse